CONTRIBUTING.md

```diff
@@ -118,6 +118,75 @@ private:
+| [Irish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ga) | None | NA |
+| [Italian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=it) | Antonio Rosella | arosellaATyahooDOTcom |
+| [Japanese](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ja) | Hironori Fujimoto (RatArmy) | broadbarredfirefishATgmailDOTcom |
+| [Korean](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ko) | mistzone | drier22ATgmailDOTcom |
+| [Latin](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=la) | None | NA |
+| [Latvian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=lv) | None | NA |
+| [Lithuanian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=lt) | Andrius Štikonas | stikonasATgmailDOTcom |
+| [Macedonian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=mk) | None | NA |
+| [Marathi](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=mr) | None | NA |
+| [Norwegian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=nb_NO) | None | NA |
+| [Old English (Latin chars)](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ang@latin) | None | NA |
+| [Polish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=pl) | ForPeace | [forum](https://forums.wesnoth.org/viewtopic.php?f=7&t=3796) |
+| [Portuguese](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=pt) | trewe | sjrs456ATyahooDOTfr |
+| [RACV Language](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=racv) | None | NA |
+| [Romanian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ro) | None | NA |
+| [Russian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ru) | Artem Khrapov (kabachuha) | artemkhrapov2001ATyandexDOTru |
+| [Scottish Gaelic](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=gd) | GunChleoc | fiosAIGforamnagaidhligDOTnet |
+| [Serbian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=sr) | None | NA |
+| [Slovak](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=sk) | Aceman | NA |
+| [Slovenian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=sl) | None | NA |
+| [Spanish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=es) | Pepe | donpepe1963ATgmailDOTcom |
+| [Spanish (Latin American)](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=es_419) | None | NA |
+| [Swedish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=sv) | Alex Alowersson (fluxbird) | alexalowersonATgmailDOTcom |
+| [Turkish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=tr) | Nilgün Belma Bugüner | nilgunATbelgelerDOTorg |
+| [Ukrainian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=uk) | robson | arobsonATyandexDOTru |
```

Comment:
https://r.wesnoth.org/p627608

I'm not sure who did the update for 1.14.6, but Robson hasn't been on the forums since 2013.
https://forums.wesnoth.org/memberlist.php?mode=viewprofile&u=129591